taf/tools/dependencies/__init__.py

```diff
@@ -49,9 +49,11 @@ def add_dependency_command():
     @click.option("--keystore", default=None, help="Location of the keystore files")
     @click.option("--prompt-for-keys", is_flag=True, default=False, help="Whether to ask the user to enter their key if not located inside the keystore directory")
     @click.option("--no-commit", is_flag=True, default=False, help="Indicates that the changes should not be committed automatically")
+    @click.option("--keys-description", help="A dictionary containing information about the "
+                  "keys or a path to a json file which stores this information")
```

Comment:
question: what's the idea behind passing `keys_description` to e.g. `dependencies` and adding key_names to auth_repo here and all other subcommands?

Reply:
The key names specified in that file will be used when the user is prompted to enter their YubiKeys, regardless of what role and why they need to sign

Reply:
The key names specified in that file will be used when the user is prompted to enter their YubiKeya, regardless of what role and why they need to sign